src/components/views/EditorSimulation/EditorSimulation.module.css

```
@@ -77,15 +73,14 @@
                 bottom: 2rem;
                 right: 2rem;
                 z-index: 1;
-                background: #211611;
+                background: var(--bg-elevated);
                 border-radius: 5rem;
                 border: 1pt solid transparent;
```

Stylelint (17.11.1)

[error] Duplicate property "border" (declaration-block-no-duplicate-properties)

(declaration-block-no-duplicate-properties)